### PR TITLE
finish product details page

### PR DESCRIPTION
**Pull request by GCyang08**
Finish product details page layout. I just need to seperate them into different components just to make it so each component has clean written code. I also renamed some components and re-designed some layouts on each category pages.

### CODE REVIEW

File: src/components/pages/ProductsDetail.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import classes from "./ProductsDetail.module.css";
+
+import Wrapper from "../UI/Wrapper";
+import Grid from "../UI/Grid";
+import ProductDetailsImages from "../ProductDetail/ProductDetailsImages";
+
+const ProductsDetail = (props) => {
+  const dummyData = {
```
Comment (arlt209) [inline src/components/pages/ProductsDetail.js:8]: yup this is good, when you add redux, you will get all of this data from redux.

File: src/components/pages/ProductsDetail.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      </Grid>
+      <div className={classes["product-details"]}>
+        <div className={classes["product-details__feautures"]}>
+          <h3>FEAUTURES</h3>
+          <p>{dummyData.features}</p>
+        </div>
+        <div className={classes["product-details__items"]}>
+          <h3>IN THE BOX</h3>
+          <div className={classes["product-details__items-list"]}>
+            {dummyData.includes.map((item, id) => {
```
Comment (arlt209) [inline src/components/pages/ProductsDetail.js:90]: nice use of includes here

File: src/components/pages/Earphones.js [hunk 3]
```diff
@@ -21,12 +21,12 @@ const Headphones = (props) => {
     <div>
       <CategoryHeader>EARPHONES</CategoryHeader>
       {earphonesData.map((earphone, id) => (
-        <CategoryGrid key={earphone.id} categoryId={id}>
+        <Grid key={earphone.id} id={id}>
           <CategoryImages categoryName={earphone.name.toUpperCase()} />
           <div>
```
Comment (arlt209) [inline src/components/pages/Earphones.js:26]: does this empty div need to be here?   if it does no worries

Comment (arlt209) [inline src/components/pages/Earphones.js:26]: does this empty div need to be here?

Comment (GCyang08) [reply src/components/pages/Earphones.js:26]: No it doesn't. That's something I'll need to fix

**Pull request merged**